Userland/Libraries/LibWeb/CSS/Parser/Tokenizer.cpp

```diff
@@ -356,18 +356,17 @@ Token Tokenizer::create_value_token(Token::Type type, DeprecatedString value)
 {
     Token token;
     token.m_type = type;
-    token.m_value = move(value);
+    token.m_value = FlyString::from_utf8(value.view()).release_value_but_fixme_should_propagate_errors();
     return token;
 }
 
 Token Tokenizer::create_value_token(Token::Type type, u32 value)
 {
     Token token = {};
     token.m_type = type;
-    // FIXME: Avoid temporary StringBuilder here
```

Comment:
pretty sure the StringBuilder is now dead code?

Reply:
:man_facepalming: